tests/bulk_create/models.py

```diff
@@ -141,3 +141,8 @@ class RelatedModel(models.Model):
     name = models.CharField(max_length=15, null=True)
     country = models.OneToOneField(Country, models.CASCADE, primary_key=True)
     big_auto_fields = models.ManyToManyField(BigAutoFieldModel)
+
+
+class Item(models.Model):
+    name = models.CharField(max_length=100)
+    last_updated = models.DateTimeField(auto_now=True)
```

Comment:
Please try to reuse and existing model, e.g. the `UpsertConflict` model above would be a good candidate.